tests/virtual_machines_test.go

```diff
@@ -32,6 +34,26 @@ var _ = Describe("Virtual Machines", func() {
 		Expect(result.Error()).NotTo(HaveOccurred())
 	})
 
+	BeforeEach(func() {
+		// Verify that there are no VMs in the cluster
+		currentVMList := &kubevirtv1.VirtualMachineList{}
```

Comment:
Not sure here, but maybe this verification should be warped in`Eventually`, because we need to wait for the controller to remove the VMs.

Reply:
This is a double check that there is no VM in the cluster, the deletion is done in AfterEach block and it is done with eventually.